components/camel-consul/src/test/java/org/apache/camel/component/consul/ConsulRegistryTest.java

```diff
@@ -49,19 +55,26 @@ public String hello(String name) {
         }
     }
 
+   
     @BeforeClass
     public static void setUp() {
-        container = ConsulTestSupport.consulContainer();
-        container.start();
+        if (DockerMachineClient.instance().isInstalled())  {
```

Comment:
What happen if I don't have docker-machine installed on my machine? I don't have docker-machine on Linux. I think we are in the same situation.

Reply:
I have tested this by removing docker machine(do not have docker at all) .it is skipping tests